mkosi/run.py

```diff
@@ -402,3 +406,44 @@ def run_workspace_command(
             if tmp.is_symlink():
                 resolve.unlink(missing_ok=True)
                 shutil.move(tmp, resolve)
+
+
+class AsyncioThread(threading.Thread):
```

Comment:
Could we name this `MkosiAsyncioThread` or `MkosiAsyncThread` so that there is no confusion if something like this ever pops up in the stdlib? The name would fit pretty well in the `asyncio` module.